CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1 @@
+Please see [our code of conduct policy](http://docs.librenms.org/General/CODE_OF_CONDUCT/).
```

Comment:
Or do you mean this new file? If it's the new file then it's part of GitHubs way to show compliance for open source projects.